dspace/config/modules/openaire-client.cfg

```diff
@@ -0,0 +1,28 @@
+#---------------------------------------------------------------#
+#----------------OPENAIRE CLIENT CONFIGURATIONS-----------------#
+#---------------------------------------------------------------#
+# These configs are used by the OPENAIRE rest client            #
+#---------------------------------------------------------------#
+
+# In order to use the API without the imposed number of requests limitation 
+# you will need to use an accessToken.
+# The accessToken it only has a validity of one hour
+# For more details about the token, please check: https://develop.openaire.eu/personalToken.html
+
+# the current OpenAIRE Rest client implementation uses basic authentication
+# Described here: https://develop.openaire.eu/basic.html
+
+# ---- Token usage required definitions ----
+# you can override this settings in your local.cfg file
+openaire.aai.tokenUsage = false
+
+# URL of the OpenAIRE authentication and authorization service 
+openaire.aai.url = https://aai.openaire.eu/oidc/token
+
+# you will be required to register at OpenAIRE (https://services.openaire.eu/uoa-user-management/registeredServices) 
+# and create your service in order to get the following data:
+openaire.aai.clientId = CLIENT_ID_HERE
+openaire.aai.clientSecret = CLIENT_SECRET_HERE
```

Comment:
Yes, OpenAIRE introduced recently a rate limitation for requests (https://develop.openaire.eu/authentication.html). We can still use their REST API but it will be a very limited usage (60 requests per hour). So I provided the two ways to support the API. For testing or sporadic usage we can ignore these settings, but advisable for a production usage. Perhaps I need to have this more clearly.

Regarding properties naming, I followed the principle of the provider in the namespace. The authentication endpoint provider isn't the same as in Rest. I thought that specifying it, those properties could be re-used for other OpenAIRE services that use the same authentication process. But, specifically for this integration, your proposal it's more suitable.